config/printer-creality-ender3s1-2022.cfg

```diff
@@ -0,0 +1,143 @@
+# This file contains pin mappings for the Creality CR6-SE with Rev. 4.5.3 Motherboard (Late 2020/2021) as the heater pins changed.
```

Comment:
Is this for Ender 3 S1 or CR6-SE?

Reply:
Ender 3 S1 but it looks like he just copy pasted the config from the CR6-SE